flink-python/pyflink/datastream/connectors/cassandra.py

```diff
@@ -0,0 +1,1648 @@
+        return self
+
+    def with_port(self, port: int) -> 'ClusterBuilder':
+        """
+        The port to use to connect to the Cassandra host.
+
+        If not set through this method, the default port (9042) will be used instead.
+        """
+        self._j_cluster_builder.withPort(port)
+        return self
+
+    def allow_beta_protocol_version(self) -> 'ClusterBuilder':
+        """
+        Create cluster connection using latest development protocol version, which is currently in
+        beta. Calling this method will result into setting USE_BETA flag in all outgoing messages,
+        which allows server to negotiate the supported protocol version even if it is currently in
+        beta.
+
+        This feature is only available starting with version V5.
+
+        Use with caution, refer to the server and protocol documentation for the details on latest
+        protocol version.
+        """
+        self._j_cluster_builder.allowBetaProtocolVersion()
+        return self
+
+    def with_max_schema_agreement_wait_seconds(self, max_schema_agreement_wait_seconds: int) \
+            -> 'ClusterBuilder':
+        """
+        Sets the maximum time to wait for schema agreement before returning from a DDL query.
+
+        If not set through this method, the default value (10 seconds) will be used.
+        """
+        self._j_cluster_builder.withMaxSchemaAgreementWaitSeconds(max_schema_agreement_wait_seconds)
+        return self
+
+    def with_protocol_version(self, version: ProtocolVersion) -> 'ClusterBuilder':
+        """
+        The native protocol version to use.
+        """
+        self._j_cluster_builder.withProtocolVersion(version._to_j_protocol_version())
+        return self
+
+    def add_contact_point(self, address: Union[str, EndPoint]) -> 'ClusterBuilder':
+        """
+        Adds a contact point by str or EndPoint.
+        """
+        if isinstance(address, str):
+            self._j_cluster_builder.addContactPoint(address)
+        else:
+            self._j_cluster_builder.addContactPoint(address._j_end_point)
+        return self
+
+    def add_contact_points(self, *addresses: Union[str, InetAddress]) -> 'ClusterBuilder':
+        """
+        Adds contact points.
+        """
+        gateway = get_gateway()
+        j_addresses = []
+        for address in addresses:
+            if isinstance(address, str):
+                j_addresses.append(address)
+            else:
+                j_addresses.append((address._j_inet_address))
+        if len(addresses) > 0:
+            if isinstance(addresses[0], str):
+                self._j_cluster_builder.addContactPoints(to_jarray(gateway.jvm.String, addresses))
+            else:
+                self._j_cluster_builder.addContactPoints(
+                    gateway.jvm.java.net.InetAddress, addresses)
+        return self
+
+    def add_contact_points_with_ports(self, *addresses: InetSocketAddress) -> 'ClusterBuilder':
```

Comment:
Could we let users specify string values? E.g. add_contact_points_with_ports(self, *addresses: str). Here, we could ensure that all the addresses should be the following format: "ip:port".

Reply:
This is the part about ClusterBuilder, I will create a new PR and fix it for easy review~